cypress/e2e/shared/flows.test.ts

```diff
@@ -239,10 +239,6 @@ describe('Flows', () => {
       .should('be.visible')
     cy.getByTestID('table-cell cool').should('not.exist')
 
-    // This is a random validator that the autorefresh option doesn't pop up
-    // In Flows again without explicit changes
-    cy.getByTestID('autorefresh-dropdown--button').should('not.exist')
```

Comment:
we might want to update this to account for the new auto refresh work, but we'd need to set the feature flag for that at the outset of the test

Reply:
do we want to test if the new auto refresh work here? or we just check if the new auto refresh button exists? 

i think we just want to check if the new button exists. it's better to test the functionality in another PR since this PR mainly removes the old auto refresh button.

Reply:
Just the button's existence, we can always work on adding some tests for the refresh functionality in the notebook's context later on